templates/rule.erb

```diff
@@ -18,6 +18,17 @@
     end
   end
 
+  if scope.to_hash.has_key?('_su')
```

Comment:
This part looks a bit overkill, why not just doing something like:
<% if @su %>
su <%= @su_owner %>  <%= @su_group %> 
<% end>
without the need to set a @_su variable.
(am I missing something?)